\begin{center}
  \rmfamily
  \setstretch{1.0}
  \tikzset{>=stealth}
  \tikzset{plasmid lines/.style={xscale = 0.2, yscale = 0.2, very thick}}
  \tikzset{legende/.style={font=\scriptsize, color=Gray, align=right}}

  \tikzset{
    plasmid/.pic={
	    \draw[Green, plasmid lines, semithick, rounded corners = 0.5ex] (3, -1) rectangle (11, 0);
	    \draw[Cerulean, plasmid lines ] (4, 0) -- (6, 0);
	    \draw[Gold1 , plasmid lines] (6,  0) -- (6.2, 0);
	    \draw[Gold1 , plasmid lines] (6,  0) -- (6.2, 0);
	    \draw[Orchid3 , plasmid lines] (6.2,  0) -- (7.8, 0);
	    \draw[SteelBlue1 , plasmid lines] (7.8, 0) -- (8.0, 0);
	    \draw[SteelBlue1 , plasmid lines] (7.8, 0) -- (8.0, 0);
	    \draw[Orange1 , plasmid lines] (8.0, 0) -- (9.8, 0);
	    \draw[Brown1 , plasmid lines] (9.8, 0) -- (10.0, 0);
    }
  }
  \tikzset{
    construct/.pic={
        \draw[plasmid lines, Cerulean] (4, 0) -- (6, 0);
        \draw[plasmid lines, Gold1] (6, 0) -- (6.2, 0);
        \draw[plasmid lines, Gold1] (6, 0) -- (6.2, 0);
        \draw[plasmid lines, Orchid3] (6.2, 0) -- (7.8, 0);
        \draw[plasmid lines, SteelBlue1] (7.8, 0) -- (8.0, 0);
        \draw[plasmid lines, SteelBlue1] (7.8, 0) -- (8.0, 0);
        \draw[plasmid lines, Orange1] (8.0, 0) -- (9.8, 0);
        \draw[plasmid lines, Brown1] (9.8, 0) -- (10.0, 0);
        \draw[plasmid lines, Green, thick] (2, 0) -- (4, 0);
        \draw[plasmid lines, Green, thick] (10, 0) -- (12, 0);
    }
  }

  \tikzset{
    gene interet/.pic={
      \draw[xscale = 0.5, Cerulean] (0, 0) -- (1, 0);
      \draw[xscale = 0.5, Gold1] (0.8, 0) -- (1, 0);
    }
  }

  \begin{tikzpicture}[scale=0.7, line width = 2pt, join=round]
    \begin{scope}[shift={(-3.5, 12)}]
      \draw[Cerulean] (0, 0) -- (1, 0);
      \draw[Gold1] (0.8, 0) -- (1, 0);
      \draw[Gray] [->] (1.5, 0) -- (2.7, 0) node[below, midway, font=\scriptsize] {PCR};
    \end{scope}

    \begin{scope}[shift={(-0.3, 10)}]
	    % \draw[Green] (0, 0) arc (0:30:3);
	    \draw[Gray]  (1, 0.5) node {\scriptsize +};
	    \draw[Green] (0, 0) arc (150:390:1);
	    \draw[Green] (1, -0.5) node {\scriptsize pGEM-T};
	    \draw[Green] (0.1, 0.3) node {\tiny \sffamily T};
	    \draw[Green] (1.7, 0.3) node {\tiny \sffamily T};
    \end{scope}


    \begin{scope}[shift={(0, 11)}, line width = 0.5 pt]
	    \foreach \y in {0,0.2,...,2} {
        \draw[Cerulean] (0, \y) -- (1, \y);
        \draw[Gold1   ] (0.9, \y) -- (1, \y);
        \draw[Cerulean] (-0.1, \y) node {\tiny \sffamily A};
        \draw[Cerulean] (1.1, \y) node {\tiny \sffamily A};
	    }
	    \draw[Cerulean, align=center] (0.5, 2.5) node[font=\tiny] {Gène\\Synthétique};
      % essaie d'aligner
    \end{scope}

    \draw[Gray] [->] (3, 11) -- (4, 11) node[above, midway]
    % \node[inner sep = 0pt] (ligase) at (5, 11)
    {\includegraphics[width=0.1\textwidth, angle = 90]{schema/construct-img/ligation.png}}
    node[below, midway, font=\scriptsize] {Ligature};
    % \draw

    \begin{scope}[shift={(5.5, 11.5)}]
	    % plasmide pGEM-T avec construction, avant infusion
	    \draw[Green] (0, 0) arc (150:390:1);
	    \draw[Brown1] (0, 0) arc (150:20:1) ;
	    \draw[Gold1] (0, 0) arc (150:30:1);
	    \draw[Cerulean] (0, 0) arc (150:40:1) ;
    \end{scope}


    \draw[Gray] [->] (8, 10) -- (10, 9.5) node[midway, above, font=\scriptsize] {Digestion};

    \begin{scope}[shift={(7, 8.5)}]
	    % plasmide ouvert, avec ancre et kana
	    \draw[Green] (0, 0) -- (4, 0) node[auto, above, midway] {\tiny{pGEM-T}};
	    \draw[Cerulean] (4, 0) -- (6, 0) node [auto, above, midway] {\tiny{Gène Synthétique}};
	    \draw[Gold1] (6, 0) -- (6.2, 0);

	    \draw[Gold1] (6, -0.3) -- (6.2, -0.3);
	    \draw[Orchid3] (6.2, -0.3) -- (7.8, -0.3) node[auto, above, midway] {\tiny{KanR}};
	    \draw[SteelBlue1] (7.8, -0.3) -- (8.0, -0.3);

	    \draw[SteelBlue1] (7.8, 0) -- (8.0, 0);
	    \draw[Orange1] (8.0, 0) -- (9.8, 0) node[auto, above, midway] {\tiny{Ancre}};
	    \draw[Brown1] (9.8, 0) -- (10.0, 0);

	    \draw[Brown1] (0, 0) -- (0.2, 0);
    \end{scope}

    \draw[Gray] [->] (14, 7.5) -- (14, 6.5) node[legende, midway, left] {Ligature \\ InFusion};

    \begin{scope}[shift={(7, 6)}]
      % Plasmide fermé avec infusion
	    \draw[Green, rounded corners, thick] (3, -0.5) rectangle (11, 0);
	    \draw[Cerulean ] (4, 0) -- (6, 0);
	    \draw[Gold1 ] (6,  0) -- (6.2, 0);
	    \draw[Gold1 ] (6,  0) -- (6.2, 0);
	    \draw[Orchid3 ] (6.2,  0) -- (7.8, 0);
	    \draw[SteelBlue1 ] (7.8, 0) -- (8.0, 0);
	    \draw[SteelBlue1 ] (7.8, 0) -- (8.0, 0);
	    \draw[Orange1 ] (8.0, 0) -- (9.8, 0);
	    \draw[Brown1 ] (9.8, 0) -- (10.0, 0);
    \end{scope}

    \draw[Gray] [->] (14, 5) -- (14, 4) node[legende, midway, left] {Amplification clonale\\dans \emph{E.coli}};

    \begin{scope}[shift={(14, 2.5)}]
      % Plasmide apmplifié avec infusion

      \matrix (A)[column sep=2mm, row sep=2mm] {
        \pic{plasmid}; & \pic{plasmid}; & \pic{plasmid}; \\
        \pic{plasmid}; & \pic{plasmid}; & \pic{plasmid}; \\
        \pic{plasmid}; & \pic{plasmid}; & \pic{plasmid}; \\
        \pic{plasmid}; & \pic{plasmid}; & \pic{plasmid}; \\
      };
    % pic {plasmid};

    \end{scope}

    \draw[Gray] [->] (14, 1) -- (14, 0) node[legende, midway, left] {Linéarisation par\\digestion enzymatique};

    \begin{scope}[shift={(14, -1)}, scale = 0.4]
      \matrix (A)[column sep=2mm, row sep=2mm] {
        \pic{construct}; & \pic{construct}; & \pic{construct}; \\
        \pic{construct}; & \pic{construct}; & \pic{construct}; \\
        \pic{construct}; & \pic{construct}; & \pic{construct}; \\
        \pic{construct}; & \pic{construct}; & \pic{construct}; \\
      };
    \end{scope}

    %% TEXTE DE DESCRIPTION
    \node[text width=8cm, legende, align = left] at (2, 3) {
      Le gène d'intérêt \tikz[baseline = -0.5ex, line width=1.3pt] {\path (0, 0)
        pic {gene interet};} est amplifié par PCR spécifique. Des bases adénines
      sont ajoutées naturellement sur les fragments au cours des derniers cycles
      de PCR. Ces fragments sont ensuite ligaturés dans le plasmide pGEM-T.
      Celui-ci est disponible ouvert avec des bases thymines sortantes. La
      ligature est catalysée par la T4DNA Ligase. Le plasmide obtenu est
      linéarisé par digestion enzymatique au site d'insertion des fragments.

      La cassette de résistance à la kanamycine \tikz[baseline = -0.5ex, very thick, scale = 0.3] {
	    \draw[Gold1] (0, 0) -- (0.2, 0);
	    \draw[Orchid3] (0.2, 0) -- (1.8, 0);
	    \draw[SteelBlue1] (1.8, 0) -- (2, 0);
      } ainsi que la région facilitant l'homologie \tikz[baseline = -0.5ex, very thick, scale = 0.3, xshift=-7.8] {
	    \draw[SteelBlue1] (7.8, 0) -- (8.0, 0);
	    \draw[Orange1] (8.0, 0) -- (9.8, 0) ;
	    \draw[Brown1] (9.8, 0) -- (10.0, 0);
      }
      sont préalablement amplifiées par PCR en utilisant des amorces porteuses de
      régions 3' flottantes, complémentaires des extrémités des fragments
      voisins. La ligature entre les trois fragments obtenus est réalisée à
      l'aide du kit Takara InFusion. Les constructions plasmidiques obtenues
      sont amplifiées clonalement par culture en milieu sélectif liquide, et
      linéarisées par digestion enzymatique.
    };


    %% Représente le génome d'acinetobacter en rouge, avec la construction
    %% associée.
    \begin{scope}[shift={(-5, -2)}]
      % représente la construction, avec l'insertion de la cassette de résistance
      \draw[Cerulean] (4, 0) -- (6, 0) node[above, midway, font=\tiny] {Gène cible};
      \draw[Gold1, ultra thick] (6, 0) -- (6.2, 0);
      \draw[Orchid3] (6.2, 0) -- (7.8, 0) node[above, midway, font=\tiny] {KanR};
      \draw[SteelBlue1, ultra thick] (7.8, 0) -- (8.0, 0);
      \draw[Orange1] (8.0, 0) -- (9.8, 0) node[above, midway, font=\tiny] {Ancre};
      \draw[Brown1, ultra thick] (9.8, 0) -- (10.0, 0);
      \draw[Green, thick] (2, 0) -- (4, 0);
      \draw[arrows = {-Stealth[left]}, Green, thick] (10, 0) -- (12, 0) node[font = \tiny, above, align = right] {Construction \\ plasmidique};

      % Génome acinetobacter
      % brin 5' 3'
      \draw[Cerulean] (5, -1) -- (7, -1);
      \draw[Orange1] (7.0, -1) -- (8.8, -1);
      \draw[SteelBlue1, thick] (2, -1) -- (5, -1);
      \draw[arrows = {-Stealth[left]}, SteelBlue1, thick] (8.8, -1) -- (12, -1) node[font=\tiny, above, align = right] {Génome \\ \emph{Acinetobacter}};
      %amorces
      \draw[Red3, ultra thick] (4, -1) -- (4.2, -1) node[above left, font=\tiny] {Amorce};
      \draw[Cerulean] (5, -1.2) -- (7, -1.2);
      \draw[Orange1] (7.0, -1.2) -- (8.8, -1.2);
      \draw[arrows = {Stealth[right, swap]-}, SteelBlue1, thick] (2, -1.2) -- (5, -1.2);
      \draw[SteelBlue1, thick] (8.8, -1.2) -- (12, -1.2);

      % Lignes reliant les fragments
      \draw[Cerulean, ultra thin] (4, 0) -- (5, -1);
      \draw[Cerulean, ultra thin] (6, 0) -- (7, -1);
      \draw[Orange1, ultra thin] (8, 0) -- (7, -1);
      \draw[Orange1, ultra thin] (9.8, 0) -- (8.8, -1);

      % petit éclair pour marquer la cassure
      \node[Gold4, font=\large] at (7, -1) {\Lightning};
      \node[Gray, font=\tiny] at (7, -1.4) {Cassure Double Brin};

      \draw[Gray] [->] (7, -1.6) -- (7, -2.4) node[legende, left, midway] {Résection};

      % brin ->
      \draw[Cerulean, arrows = {-Stealth[left, scale = 1]}, very thick] (5, -2.5) -- (7, -2.5);
      \draw[Orange1] (8.3, -2.5) -- (8.8, -2.5);
      \draw[SteelBlue1, thick] (2, -2.5) -- (5, -2.5);
      \draw[arrows = {-Stealth[left]}, SteelBlue1, thick] (8.8, -2.5) -- (12, -2.5);
      % brin <-
      \draw[Cerulean, ultra thick] (5, -2.7) -- (5.5, -2.7);
      \draw[arrows = {Stealth[right, swap, scale = 1]-}, Orange1, very thick] (7.0, -2.7) -- (8.8, -2.7);
      \draw[arrows = {Stealth[right, swap]-}, SteelBlue1, thick] (2, -2.7) -- (5, -2.7);
      \draw[SteelBlue1, thick] (8.8, -2.7) -- (12, -2.7);

      \draw[Gray] [->] (7, -2.8) -- (7, -3.4) node[legende, midway, left] {Réparation};

      \draw[Cerulean, arrows = {-Stealth[left, scale = 1]}, very thick] (5, -3.5) -- (7, -3.5);
      \draw[Orange1] (8.3, -3.5) -- (8.8, -3.5);
      \draw[SteelBlue1, thick] (2, -3.5) -- (5, -3.5);
      \draw[arrows = {-Stealth[left]}, SteelBlue1, thick] (8.8, -3.5) -- (12, -3.5);
      % brin <-
      \draw[Cerulean, ultra thick] (5, -3.7) -- (5.5, -3.7);
      \draw[ Orange1, very thick] (8.8, -3.7) -- ++(-0.5, 0) -- ++(-0.2, -0.6) -- ++(-0.7, 0);
      \draw[arrows = {Stealth[right, swap]-}, SteelBlue1, thick] (2, -3.7) -- (5, -3.7);
      \draw[SteelBlue1, thick] (8.8, -3.7) -- (12, -3.7);

      %% fragment insert avec boucle pour cassette de résistance
      % je place l'origine de la construction au début da la région ancre sur le
      % génome d'acinetobacter. C'est plus facile pour s'aligner
      \draw[Orange1, very thick]    (8.8, -4.6) -- ++(-2, 0);
      \draw[SteelBlue1, very thick] (6.8, -4.6) -- ++(-0.2, 0);
      \draw[Orchid3, very thick]    (6.6, -4.6) -- ++(-2, 0);
      \draw[Gold1, very thick]      (4.6, -4.6) -- ++(-0.2, 0);
      \draw[Cerulean, very thick]   (4.4, -4.6) -- ++(-2, 0);
      \draw[Green, thick] (2.4, -4.6) -- ++(-0.4, 0);
      \draw[Green, thick, arrows = {-Stealth[left]}] (8.8, -4.6) -- ++(2, 0);

      %% resynthèse
      \draw[densely dotted, Orange1, very thick]    (7.35, -4.3) --  (6.8, -4.3);
      \draw[densely dotted, SteelBlue1, very thick] (6.8, -4.3) -- ++(-0.2, 0);
      \draw[densely dotted, Orchid3, very thick]    (6.6, -4.3) -- ++(-2, 0);
      \draw[densely dotted, Gold1, very thick]      (4.6, -4.3) -- ++(-0.2, 0);
      \draw[densely dotted, Cerulean, very thick, arrows = {-Stealth[left, scale = 0.5]}]   (4.4, -4.3) -- ++(-2, 0);

    \end{scope}

    %% Représente la portion du génome d'acinetobacter qui est résectée après la
    %% cassure.
    % \begin{scope}[shift={(-5, -5)}]
    %   \draw (0, 1) -- (2, 1);
    % \end{scope}

  \end{tikzpicture}


\end{center}
